gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -92,6 +96,7 @@ class BaselineIdea extends AbstractBaselinePlugin {
             addGitHubIssueNavigation(node)
             ignoreCommonShadedPackages(node)
         }
+        configureProjectForIntellijImport(project)
```

Comment:
move this inside `applyToRootProject`, since this configuration should only happen once for the whole repo, not once per gradle project

Reply:
nvm, tricked by hidden code in the UI :(